modules/local/bowtie2_assembly_align.nf

```diff
@@ -0,0 +1,44 @@
+//https://github.dev/nf-core/mag
```

Comment:
oh great doc, thanks for adding this

Reply:
I'd add this comment to the file itself and point out where it differs from the nf-core version